flowblade-trunk/Flowblade/sequence.py

```diff
@@ -268,6 +268,13 @@ def _add_track_attributes(self, track, type):
         # Tracks may be FREE or LOCKED
         track.edit_freedom = FREE
 
+    def get_track_counts(self):
+        """returns (video track count, audio track count)"""
+        nv = len(self.tracks) - self.first_video_index - 1
+        na = self.first_video_index - 1
+        print ("get_tracks_count", self.first_video_index, len(self.tracks), nv, na)
```

Comment:
Sorry, I didn't mean to leave this print statement here.  Harmless, but needless noise

Reply:
No, prob I can remove next time I do a coding session.